src/infrastructure/jira/jira-client/jira-client.ts

```diff
@@ -109,15 +110,17 @@ class JiraClient {
 		connectInstallation: ConnectInstallation,
 	): Promise<GetIssuePropertyResponse> => {
 		const url = new URL(
-			`/rest/api/2/issue/${issueIdOrKey}/properties/${propertyKey}`,
+			`${ISSUE_PROPERTY_BASE_URI}/${issueIdOrKey}/properties/${propertyKey}`,
```

Comment:
**nitpick:** Technically, `ISSUE_PROPERTY_BASE_URI` is not a [URI](https://datatracker.ietf.org/doc/html/rfc3986) but a path. Therefore, consider renaming it to `ISSUE_PROPERTY_API_PATH`. This will also help to avoid confusion with `connectInstallation.baseUrl`.

Also, we could just inline the constant value into the string (`/rest/api/2/issue/${issueIdOrKey}/properties/${propertyKey}`) for the following reason:

- It gets easier to understand what endpoint it refers to since we don't need to jump among multiple places in the code.
- It is used just a few times within `JiraClient` and it neither a magic "string" nor subject to change. Therefore, it is perfectly fine to use it as is.

Reply:
Removed `ISSUE_PROPERTY_BASE_URI` and just put the string in directly, I agree that there's no benefit to pulling it out.